src/lib/services/pat-service.ts

```diff
@@ -33,24 +33,24 @@ export default class PatService {
         this.eventStore = eventStore;
     }
 
-    async createPat(pat: IPat, user: User): Promise<IPat> {
-        await this.validatePat(pat, user.id);
+    async createPat(pat: IPat, userId: number, creator: User): Promise<IPat> {
```

Comment:
So, the userId is the target of the PAT (or will be the owner), while the creator can be a different person, right? I think this is when an administrator creates a token on behalf of a user (who might not have permission to create it themselves).

Nitpicking, maybe a small rename in the variable can give more clarity

```suggestion
    async createPat(pat: IPat, forUserId: number, creator: User): Promise<IPat> {
```